cfg/admin.txt

```diff
@@ -1,125 +1 @@
 jayehh role=admin
-optimumtact role=admin
-newsta role=admin
-expletives role=admin
-kingofkosmos role=admin
-mrstonedone role=admin
-microscopics role=admin
-gunhog role=admin
-korphaeron role=admin
-razharas role=admin
-lordpidey role=admin
-niknakflak role=admin
-rolan7 role=admin
-quarxink role=admin
-adrix89 role=admin
-tle role=admin
-xsi role=admin
-scaredofshadows role=admin
-neofite role=admin
-trubblebass role=admin
-mport2004 role=admin
-deuryn role=admin
-agouri role=admin
-errorage role=admin
-superxpdude role=admin
-petethegoat role=admin
-nodrak role=admin
-carnwennan role=admin
-ikarrus role=admin
-cheridan role=admin
-giacomand role=admin
-rockdtben role=admin
-sieve role=admin
-aranclanos role=admin
-intigracy role=admin
-dumpdavidson role=admin
-kazeespada role=admin
-malkevin role=admin
-incoming role=admin
-demas role=admin
-fleure role=admin
-ricotez role=admin
-misterperson role=admin
-crimsonvision role=admin
-iamgoofball role=admin
-zelacks role=admin
-androidsfv role=admin
-miggles role=admin
-jordie0608 role=admin
-s0ldi3rkr4s0 role=admin
-ergovisavi role=admin
-vistapowa role=admin
-miauw62 role=admin
-rumia29 role=admin
-bobylein role=admin
-sirbayer role=admin
-hornygranny role=admin
-yota role=admin
-firecage role=admin
-donkieyo role=admin
-argoneus role=admin
-paprka role=admin
-cookingboy3 role=admin
-limeliz role=admin
-steelpoint role=admin
-phil235 role=admin
-corruptcomputer role=admin
-xxnoob role=admin
-tkdrg role=admin
-cuboos role=admin
-thunder12345 role=admin
-wjohnston role=admin
-mandurrh role=admin
-thurgatar role=admin
-xerux role=admin
-dannno role=admin
-lo6a4evskiy role=admin
-vekter role=admin
-ahammer18 role=admin
-account12 role=admin
-fayrik role=admin
-shadowlight213 role=admin
-drovidicorv role=admin
-dunc role=admin
-mmmiracles role=admin
-bear1ake role=admin
-coreoverload role=admin
-jalleo role=admin
-changelingrain role=admin
-foxpmccloud role=admin
-xhuis role=admin
-astralenigma role=admin
-tokiko1 role=admin
-supersayu role=admin
-lzimann role=admin
-as334 role=admin
-neersighted role=admin
-swankcookie role=admin
-ressler role=admin
-folix role=admin
-bawhoppennn role=admin
-anturke role=admin
-lumipharon role=admin
-bgobandit role=admin
-coiax role=admin
-randommarine role=admin
-pkpenguin321 role=admin
-technoalchemist role=admin
-aloraydrel role=admin
-quiltyquilty role=admin
-snipedragon role=admin
-fjeld role=admin
-kevinz000 role=admin
-tacolizard role=admin
-trustygun role=admin
-cyberboss role=admin
-pjb3005 role=admin
-sweaterkittens role=admin
-feemjmeem role=admin
-jstheguy role=admin
-excessiveuseofcobby role=admin
-plizzard role=admin
-octareenroon91 role=admin
-serpentarium role=admin
-cebutris role=admin
```

Comment:
This entire file does basically nothing on a live server. There's no need to axe it.

Reply:
m e h